.github/workflows/update-lock-files.yml

```diff
@@ -28,6 +28,8 @@ jobs:
           - name: pypy
             update_script_args: "--select-tag pypy"
             additional_commit_message: "[pypy]"
+          - name: array API
+            update_scrupt_args: "--select-tag gpu"
```

Comment:
Let's make it explicit that this is about CUDA here as well.

```suggestion
          - name: cuda-gpu
            update_script_args: "--select-tag cuda"
```

Reply:
I am wondering if we could find a way to get a GPU workflow dispatch to automatically run on the update lock files PR opened by the bot.

Otherwise the person reviewing the GPU lock file update PRs will have to remember to always perform a manual dispatch on those PRs before merging them.

Maybe it's possible to put the `scikit-learn-bot` fork in some kind of allow-list in the in the `on/pull_request` trigger configuration itself.

Alternatively, we could use a generic `on/pull_request` trigger and do some filtering in the `jobs/tests/if` clause as documented in:

https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#accessing-and-using-event-properties

Hoping that the GPU VM is not spawned just to evaluate the `jobs/tests/if` conditions.

Reply:
I think we could add a `gh workflow run .github/workflowcuda-gpu-ci.yml PR_ID` to the workflow file of the lock file update bot. That way the workflow would run on the PR that the bot created.

Reply:
Indeed that sounds like an excellent solution.